lib/bluepill/controller.rb

```diff
@@ -86,6 +86,7 @@ def grep_pattern(application, query = nil)
     def cleanup_bluepill_directory
       running_applications.each do |app|
         pid = pid_for(app)
+        next unless pid
         next if pid || System.pid_alive?(pid)
```

Comment:
If I’m reading this correctly, these three lines say:
1. `next` if there is a pid for the app
2. `next` if there is not a pid for that app

That logic can’t be right, can it?